### DISCUSSION TITLE
Contributions welcomed

### DISCUSSION THREAD

**Discussion by prakkritti1999** [community/community#144661] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

hey folks,
I am open to contributions and suggestions to my self-driven Spring Boot based project named MeTime. Here is the repository : https://github.com/prakkritti1999/MeTime---One-Stop-Travel-Solution.git

**Comment by prothesbarai** [comment]
Response #1
